xwiki-platform-core/xwiki-platform-extension/xwiki-platform-extension-handlers/xwiki-platform-extension-handler-xar/src/main/resources/ApplicationResources.properties

```diff
@@ -63,7 +63,7 @@ extension.xar.log.delete.document=Deleted document [{}]
 extension.xar.log.delete.document.failure=Failed to delete document [{}]
 
 extension.xar.question.defaultConflictAction.label=Default conflict answer
-extension.xar.question.defaultConflictAction.hint=When dealing with XAR extensions it's possible to get conflicts (most of the time caused by changes made locally to documents belonging to extensions). When that happens, Extension Manager will try to choose the safest way to workaround the conflict but you can change the behavior using the following options.
+extension.xar.question.defaultConflictAction.hint=When dealing with XAR extensions it is possible to get conflicts (most of the time caused by changes made locally to documents belonging to extensions). When that happens, Extension Manager will try to choose the safest way to workaround the conflict but you can change the behavior using the following options.
 
```

Comment:
Please explain why you consider this is a mistake. The apostrophe, in this case, marks an omission of the `i` from `it is` to `it's` and it's a perfectly valid use of English language.

This issue does not seem valid.